src/interfaces/IEthMultiVault.sol

```diff
@@ -531,6 +542,19 @@ interface IEthMultiVault {
     ///       if the vault ID does not exist/is not an atom.
     function depositAtom(address receiver, uint256 id) external payable returns (uint256);
 
+    /// @notice deposit eth into an atom vault and grant ownership of 'shares' to 'reciever'
```

Comment:
3.10 - New *Curve Functions in EthMultiVault Missing From Interface

We have added the new curve related methods to the interface here.